backend/src/routes/api/modelRegistries/modelRegistryUtils.ts

```diff
@@ -1,11 +1,16 @@
-import { MODEL_REGISTRY_NAMESPACE } from '../../../utils/constants';
 import { KubeFastifyInstance, ModelRegistryKind, RecursivePartial } from '../../../types';
 import { PatchUtils, V1Secret, V1Status } from '@kubernetes/client-node';
+import { getClusterStatus } from '../../../utils/dsc';
 
 const MODEL_REGISTRY_API_GROUP = 'modelregistry.opendatahub.io';
 const MODEL_REGISTRY_API_VERSION = 'v1alpha1';
 const MODEL_REGISTRY_PLURAL = 'modelregistries';
 
+export const getModelRegistryNamespace = async (fastify: KubeFastifyInstance): Promise<string> => {
+  const modelRegistryNamespace = await getClusterStatus(fastify);
```

Comment:
In case you look at this @andrewballantyne - I considered giving the feedback here based on our discussion that we should instead add a `resourceWatcher` to `backend/src/utils/resourceUtils.ts` and use that, but I don't think that will work because we need this resolved at proxy registration time and the initial fetch of the watchers may not be complete by then (if I understand it correctly).

For the regular user's consumption of the MR API via `proxyService`, we'll only fetch the namespace once (at proxy registration time), so that part is fine. However, as implemented, we will be fetching the namespace each time the admin lists/creates/deletes model registries. Perhaps we could also just await it before calling the functions to register routes in `backend/src/routes/api/modelRegistries/index.ts`. @ppadti that might be worth trying - await getting the namespace once when we set up those routes (before calling `fastify.get/post/patch/delete`) and pass the string namespace into the utilities below (`listModelRegistries`, etc) instead of awaiting the namespace in those utils. That'll save us from having an extra network call for the namespace on every interaction in the settings page.